coalib/bearlib/languages/documentation/DocumentationExtraction.py

```diff
@@ -250,4 +251,58 @@ def extract_documentation_with_markers(content, docstyle_definition):
             marker_dict,
             docstyle_definition)
         if doc:
+            # Instantiate padding
+            top_padding = 0
+            bottom_padding = 0
+            start_index = doc.range.start.line - 2
+            end_index = doc.range.end.line
+            while start_index >= 0 and content[start_index] == '\n':
+                top_padding += 1
+                start_index -= 1
+            # If the end_index is instantiated above the len(content) i.e.
+            # In case where ending marker of docstring is at the last line.
+            # Then the bottom_padding will be default to 0. This will also
+            # prevent IndexError raised by content[end_index].
+            while end_index < len(content) and content[end_index] == '\n':
+                # This condition will take place if theres an inline docstring
+                # following documentation.
+                if ((doc.marker[2]+'\n') != content[end_index-1][-4:]
+                        and bottom_padding == 0):
+                    break
+                bottom_padding += 1
+                end_index += 1
+            doc.top_padding = top_padding
+            doc.bottom_padding = bottom_padding
```

Comment:
You could remove these two lines by replacing all occurrences of `top_padding` with `doc.top_padding` and dito for `bottom_padding`. There's no real need for a temporary variable.